docs/releases/v2015.2.rst

```diff
@@ -1,17 +1,31 @@
 Gluon 2015.2 (in development)
 =============================
 
-Added (and removed) hardware support
+Added hardware support
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ar71xx-generic
+^^^^^^^^^^^^^^
 
+* TP-Link
+
+  - CPE 210/220/510/520 (v1.1)
+  - TL-WR841N/ND (v10)
+  - TL-WR901ND (v1)
+  - TL-WR940N (v3)
+  - TL-WR941N (v6)
+  
+* Ubiquity
+
+  - Air Gateway
 
 New features
 ~~~~~~~~~~~~
-
+802.11s can now be used as an alternative to IBSS mode (ad-hoc). The statuspage has been reworked completely. French language support has been added. The numbers of clients connected are announced per band now (2.4/5GHz).
```

Comment:
I think you can leave the "New features" and "Bugfixes" sections to us. We want the explanations to be a bit more detailed.